### PR TITLE
PLATEXP-1735: Modified reset-user-password-on-active-directory-domain

### PR DESCRIPTION
**Pull request by asmita32675**
@ssalinas24 Please review the article.
https://rackspace.atlassian.net/browse/PLATEXP-1735

### CODE REVIEW

File: content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md [hunk 1]
```diff
@@ -16,29 +16,33 @@ Microsoft&reg; Active Directory user in the MyRackspace portal.
...
 
-1. Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with
+**Step 1**. Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with
    your username and password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="Enter login credentials" src="/support/how-to/reset-user-password-on-active-directory-domain/resetpassword1.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+**Step 2.** In the top navigation bar, choose **Support** > **Support Center**.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md:24]: Done

File: content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md [hunk 2]
```diff
@@ -16,29 +16,33 @@ Microsoft&reg; Active Directory user in the MyRackspace portal.
...
 
-4. Select the **Subject** field.
+   <img width="941" alt="Select support center from drop down menu" src="/support/how-to/reset-user-password-on-active-directory-domain/resetpassword2.png">
 
-5. From the drop-down menu, select **Reset User Password on Active Directory Domain**.
+**Step 3.** Under **Common Request** menu, click **See All Templates**. The **See All** Templates displays list of all templates.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md:28]: done.

File: content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md [hunk 3]
```diff
@@ -16,29 +16,33 @@ Microsoft&reg; Active Directory user in the MyRackspace portal.
...
 
-    {{<image src="password1.png" alt="" title="">}}
+   <img width="958" alt="Displays list of templates" src="/support/how-to/reset-user-password-on-active-directory-domain/resetpassword3.png">
 
-6. Enter the **Username** and **Domain** of the user account for which you want
+**Step 4.** From the **Common Request** menu, select **Reset User Password on Active Directory Domain** as shown in following image.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md:32]: Done

File: content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md [hunk 4]
```diff
@@ -16,29 +16,33 @@ Microsoft&reg; Active Directory user in the MyRackspace portal.
...
    After the request completes, the system generates a secure password
    automatically for this user and provides it in the ticket. You can change
    this password later.
 
-7. Select **Create Ticket**.
+**Step 6.** Click **Submit** button.
 
-   After the ticket generates, you see your ticket list in the MyRackspace Portal.
+   After the ticket generates, you see the ticket list in the MyRackspace Portal.
    It typically takes a few minutes for the ticket creation process to complete.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md:46]: Done

**Review by ssalinas24 — APPROVED** [review]

**Pull request merged**